docs/tools/kompose/user-guide.md

```diff
@@ -427,13 +418,17 @@ If you want to create normal pods without controllers you can use `restart` cons
 
 **Note**: controller object could be `deployment` or `replicationcontroller`, etc.
 
-For e.g. `mariadb` service will become pod down here.
+For e.g. `pival` service will become pod down here. This container calculated value of `pi`.
 
 ```yaml
-version: "2"
+version: '2'
 
 services:
-  mariadb:
-    image: centos/mariadb
-    restart: "no"
+  pival:
+    image: perl
+    command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+    restart: "on-failure"
 ```
+
+#### Notes:
+-- If the Docker Compose file has a volume specified for a service, the Deployment (Kubernetes) or DeploymentConfig (OpenShift) strategy is changed to "Recreate" instead of "RollingUpdate" (default). This is done to avoid multiple instances of a service from accessing a volume at the same time.
```

Comment:
What's the difference between this **Notes** and the above **Note** (level-wise)?